python/fastquant/strategies/base.py

```diff
@@ -289,6 +294,32 @@ def next(self):
                         self.sell(exectype=bt.Order.StopTrail, trailpercent=self.stop_trail, size=final_size)
 
         elif self.sell_signal():
+            if self.allow_short == True:
+
+                # Sell short based on the closing price of the previous day
+                if self.execution_type == "close":
+
+                    if self.transaction_logging:
+                        self.log("SELL CREATE, %.2f" % self.dataclose[0])
+
+                    sell_prop_size = int(SELL_PROP * 
+                                         self.broker.getvalue() / 
+                                         self.dataclose[0])
+                    self.order = self.sell(size=sell_prop_size)
+                
+                 # Buy based on the opening price of the next closing day (only works "open" data exists in the dataset)
+                else:
+
+                    if self.transaction_logging:
+                        self.log("SELL CREATE, %.2f" % self.dataclose[1])
+                    
+                    sell_prop_size = int(self.short_max *
+                                         self.cash /
+                                         self.dataclose[1] /
+                                         (1 + self.commission + 0.001))
+                    self.order = self.sell(size=sell_prop_size)
+
+
             if stock_value > 0:
```

Comment:
```suggestion
            elif stock_value > 0:
```

Reply:
Changing this to elif to avoid possibility of selling twice when both `self.allow_short == True` and `stock_value > 0`